worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -2676,10 +2799,6 @@ public boolean apply(BlockVector3 position) throws WorldEditException {
      * @return number of blocks affected
      * @throws MaxChangedBlocksException thrown if too many blocks are changed
      */
-    public int hollowOutRegion(Region region, int thickness, Pattern pattern) throws MaxChangedBlocksException {
```

Comment:
Why remove the (presumably) WorldEdit method and change behaviour/break API compact?

Reply:
Named method is called nowhere.